test/unit/test_contrbtree.c

```diff
@@ -25,15 +25,15 @@ bool test_rz_rbtree_cont_insert() {
 	RContRBTree *tree = rz_rbtree_cont_new();
 	ut32 i;
 	for (i = 0; i < 2000; i++) {
-		ut32 v = (ut32)rz_num_rand(UT32_MAX >> 1);
-		rz_rbtree_cont_insert(tree, (void *)(size_t)v, simple_cmp, NULL);
+		ut64 v = (ut64)rz_num_rand(UT64_MAX >> 1);
```

Comment:
Then it will probably needs a new test for 64bit, instead of modifying old one

Reply:
this isn't a test for random numbers if I understand it correctly. This is to test RBTree's insert implementation. Do we really need to add another insert check with 64bit random numbers?

Reply:
What I meant - keep the 32 bit implementation here, the `rz_num_rand32()`